public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
@henites72 I'm not sure if we need this file. Let me know if it's necessary.

Reply:
Yeah, we don't need it.